packages/did/src/Web3Names/Web3Names.chain.ts

```diff
@@ -95,7 +95,7 @@ export async function queryWeb3NameForDidIdentifier(
  */
 export async function queryDidIdentifierForWeb3Name(
   name: Web3Name
```

Comment:
Can we somehow enforce minimum/maximum length on strings as well? If so, that would solve a lot more issues in a more elegant way.

Reply:
I don't think there is a non-hacky way to do this, and it wouldn't replace runtime checks. So I would say no.